GetNOTAM.py

```diff
@@ -17,19 +18,24 @@
 #         finalAirLocation[0]: long for end location
 def startNotam():
 # User inputs
+    locations = []
     effectiveStartDate = input("Enter effective start date (YYYY-MM-DD HH:MM:SS): ")
     effectiveEndDate = input("Enter effective end date (YYYY-MM-DD HH:MM:SS): ")
-    location = input("Enter airport: ")
-    finalLocation = input("Enter ending airport: ")
-
-    airLocation = AirportsLatLongConverter.get_lat_and_lon(location)
-    finalAirLocation = AirportsLatLongConverter.get_lat_and_lon(finalLocation)
-
+    locations.append(input("Enter airport: "))
+    inputF = input("Would you like to have multiple end locations (Y/N)")
+    if inputF == "N":
+        locations.append(input("Enter ending airport: "))
+    else:      
+        location = input("Enter airport(Type N to stop): ")
+        while(location.upper() != "N"):

```

Comment:
a "sentinel" value like `"N"` is fine, but I'd also suggest (from a usability standpoint) to allow a empty string to be a "sentinel" value to terminate the loop